packages/quantic/cypress/page-objects/search.ts

```diff
@@ -17,16 +24,10 @@ export const InterceptAliases = {
 export function interceptSearch() {
   return cy
     .intercept('POST', '**/rest/ua/v15/analytics/*', (req) => {
```

Comment:
I refactored the UA aliases to have a direct mapping between the UA events and the aliases. It will scale better as we handle more UA events.